public/index.html

```diff
@@ -10,34 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
```

Comment:
- [ ] Kindly delete the `robots.txt` and `manifest.json` from the public directory as suggested in the previous review. unnecessary files( boilerplate that got created by the CRA (texts, images, and styles)).